BoostConfig.cmake.in

```diff
@@ -0,0 +1,21 @@
+set(_boost_deps @BOOST_FIND_PACKAGE@)
+
+foreach(dep ${_boost_deps})
+  find_package(${dep} QUIET)
```

Comment:
You're just trying to find dependencies across boost components right? (For example how `filesystem` probably requires `system`?). In that case, shouldn't you be using the `find_dependency()` macro? According to the docs, it's the job of the downstream CMake scripts to find dependencies:

https://cmake.org/cmake/help/v3.6/module/CMakeFindDependencyMacro.html#module:CMakeFindDependencyMacro

Reply:
It's for third party dependencies as you guessed later. Things like zlib, bzip2 or mpi.